biz.aQute.bndlib/src/aQute/bnd/cdi/CDIAnnotations.java

```diff
@@ -59,74 +61,79 @@ public boolean analyzeJar(Analyzer analyzer) throws Exception {
 		if (header.size() == 0)
 			return false;
 
-		Parameters optionsHeader = OSGiHeader
-			.parseHeader(analyzer.mergeProperties(Constants.CDIANNOTATIONS_OPTIONS));
-		EnumSet<Options> options = EnumSet.noneOf(Options.class);
-		for (Map.Entry<String, Attrs> entry : optionsHeader.entrySet()) {
-			try {
-				Options.parseOption(entry, options, this);
-			} catch (IllegalArgumentException e) {
-				analyzer.error("Unrecognized %s value %s with attributes %s, expected values are %s",
-					Constants.CDIANNOTATIONS_OPTIONS, entry.getKey(), entry.getValue(), EnumSet.allOf(Options.class));
-			}
-		}
-
-		if (options.isEmpty()) {
-			// set the default mode, if org.osgi.service.cdi.annotations.Bean is
-			// on the classpath set to `annotated_by_bean`, otherwise set to
-			// `none`
-
-			TypeRef typeRef = analyzer.getTypeRefFromFQN("org.osgi.service.cdi.annotations.Bean");
-			if (typeRef != null)
-				options.add(Options.annotated_by_bean);
-			else
-				options.add(Options.none);
-		}
-
-		if (options.contains(Options.none)) {
-			return false;
-		}
-
 		Instructions instructions = new Instructions(header);
 		Collection<Clazz> list = analyzer.getClassspace()
 			.values();
+		TypeRef beanTypeRef = analyzer.getTypeRefFromFQN("org.osgi.service.cdi.annotations.Bean");
```

Comment:
This variable will always be non-null since a TypeRef is always created even if the referenced class is not visible to the analyzer.